src/test/java/com/google/javascript/clutz/OptionsTest.java

```diff
@@ -46,7 +46,7 @@ public void testFilterSourcesDepgraphs() throws Exception {
             "javascript/closure/string/string.js", "blaze-out/blah/my/blaze-out-file.js")
         .inOrder();
     assertThat(opts.externs)
-        .containsExactly("javascript/common/dom.js", "extern1.js", "extern2.js")
+        .containsExactly("my/root/extern.js", "my/nonroot/extern.js", "extern1.js", "extern2.js")
```

Comment:
I don't like this test.  Its name says it's about filtering sources, but it has a bunch of other assertions.  Could you rename the test or split it up?

Reply:
Generally, random refactoring is best left for other PRs, but these are tests so pretty harmless.

Split into sources assertions and extern assertions. Output assertions are not needed (not affected by depgraph_filter_sources).